hash

```diff
@@ -55,8 +56,12 @@ sparkSubmit \
   --master "${MASTER:=local[*]}" \
   --name "${app_name}" \
   --jars "${deps_jar}" \
+  --conf "spark.driver.memory=4g" \
   --conf "spark.executor.memory=4g" \
   --conf "spark.default.parallelism=8" \
+  --conf "spark.serializer=org.apache.spark.serializer.KryoSerializer" \
```

Comment:
According to the [docs](https://spark.apache.org/docs/latest/tuning.html#data-serialization) KryoSerializer requires classes registration. Should we at least create an issue for that?

Reply:
👍 this is not mandatory but indeed will save some space later on https://github.com/src-d/gemini/issues/141